src/installation/index.md

```diff
@@ -1,61 +1,167 @@
-# Rust on ESP targets
+# Setting Up a Development Environment
 
 With an understanding of the ecosystem surrounding Rust on Espressif chips, we can move on to actual development. If you are not aware of the two possible development approaches or do not understand the differences between writing `std` and `no_std` applications, please first read the [Ecosystem Overview] chapter.
 
-Let's take a moment to discuss the Rust support for the architectures of the Espressif chips in more detail. At this moment, Espressif SoCs are based on two different architectures: `RISC-V` and `Xtensa`. The support for those two architectures in the Rust programming language is very different.
+Let's take a moment to discuss the Rust support for the different architectures of the Espressif chips and the different approaches in more detail. At this moment, Espressif SoCs are based on two different architectures: `RISC-V` and `Xtensa`. The support for those two architectures in the Rust programming language is very different.
 
-## Rust in RISC-V targets
+[Ecosystem Overview]: ../overview/index.md
+
+## RISC-V targets
+
+The `RISC-V` architecture has support in the mainline Rust compiler so, the setup is relatively simple. There are two ways of proceeding with the installation:
+- Using [`espup`, a tool that will be covered later]
+- Using the official Rust tools
+
+If you only want to use `RISC-V` targets, you can use the official Rust tools, for this approach we need [`rustup`] installed, and a [Rust nightly toolchain] with the `rust-src` [component]. We can install a nightly toolchain with the `rust-src` component via:
+
+```bash
+rustup toolchain install nightly --component rust-src
+```
+
+These are the two recommended targets for most Espressif `RISC-V` chips:
+- For bare-metal (`no_std`) applications: `riscv32imc-unknown-none-elf`
+- For applications that require `std`: `riscv32imc-esp-espidf`
+
+> #### A note in RISC-V `no_std` Rust targets.
+>
+> There are 3 different flavors of RISC-V 32 target in Rust:
+> - `riscv32i-unknown-none-elf`
+> - `riscv32imc-unknown-none-elf`
+> - `riscv32imac-unknown-none-elf`
+>
+> Where:
+> - `32i`: Base Integer Instruction Set for 32 bits
+> - `m`: Standard Extension for Integer Multiplication and Division
+> - `a`: Standard Extension for Atomic Instructions
+>   - Only available in ESP32-C6 at the moment
+> - `c`: Standard Extension for Compressed Instructions
```

Comment:
I actually think this whole section is kind of unneccessary, and we should probably just have a small blurb and then link out to some page which covers RISC-V extensions like this one:  
https://en.wikichip.org/wiki/risc-v/standard_extensions